apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -157,107 +151,10 @@ impl Drop for Telemetry {
 impl Plugin for Telemetry {
     type Config = config::Conf;
 
-    fn activate(&mut self) {
-        // The active service is about to be swapped in.
-        // The rest of this code in this method is expected to succeed.
-        // The issue is that Otel uses globals for a bunch of stuff.
-        // If we move to a completely tower based architecture then we could make this nicer.
-        let tracer_provider = self
-            .tracer_provider
-            .take()
-            .expect("trace_provider will have been set in startup, qed");
-        let tracer = tracer_provider.versioned_tracer(
-            "apollo-router",
-            Some(env!("CARGO_PKG_VERSION")),
-            None,
-        );
-        let telemetry = tracing_opentelemetry::layer().with_tracer(tracer);
-        Self::replace_tracer_provider(tracer_provider);
-
-        replace_layer(Box::new(telemetry))
-            .expect("set_global_subscriber() was not called at startup, fatal");
-        opentelemetry::global::set_error_handler(handle_error)
-            .expect("otel error handler lock poisoned, fatal");
-        global::set_text_map_propagator(Self::create_propagator(&self.config));
-    }
+    fn activate(&mut self) {}
```

Comment:
(This can be removed is it's now the same as the trait's default)

Reply:
fixed in 86b240db